README.md

```diff
@@ -40,8 +40,17 @@ integrated citation support and bibliography format conversion
 The main point of entry is the function `citeproc` from the
 module `Citeproc`.  This takes as arguments:
 
-- a `CiteprocOptions` structure (which currently just allows you
-  to set whether citations are hyperlinked to the bibliography)
+- a `CiteprocOptions` structure, which includes the following options:
+
+  * `linkCitations` controls whether citations are hyperlinked 
+     to the bibliography.
+
+  * `linkBibliography` automatically linkifies any identifiers (DOI, 
+     PMCID, PMID, or URL) appearing in a bibliography entry.  When
+     an entry has a DOI, PMCID, PMID, or URL available (in order of 
+     priority) but the style does not explicitly render at least one
```

Comment:
I wasn't sure what was meant here by "in order of priority."  Why does the order matter?  We're just checking to see if [one of these is present, but none are rendered], right?

Reply:
Should we mention or link to the CSL recommendation here?

Reply:
By "in order of priority", I mean that if an entry specifies more than one identifier (DOI, PMCID, PMID, URL), the target of the hyperlink will be chosen in that order of priority.  Is there some way I can make the wording more clear?  And sure, linking the CSL recommendation makes sense.

Reply:
I get it now.  Maybe it's okay as it stands, but perhaps clearer would be:

When an entry has a DOI, PMCID, PMID, or URL available but none of these are rendered by the style, add a link to the title (or, if no title is present, the whole entry), using the URL for the DOI, PMCID, PMID, or URL (in that order of priority).

Reply:
That is more clear, thanks!  I committed it along with a link to the spec.